core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -416,6 +417,7 @@ public PropServerConfiguration(
 
         this.mkdirMode = getInt(properties, env, PropertyKey.CAIRO_MKDIR_MODE, 509);
         this.maxFileNameLength = getInt(properties, env, PropertyKey.CAIRO_MAX_FILE_NAME_LENGTH, 127);
+        this.defaultTableWriteMode = WriteMode.valueOf(getString(properties, env, PropertyKey.CAIRO_DEFAULT_TABLE_WRITE_MODE, "direct"));
```

Comment:
is this table `write` or `writeR` mode?

Reply:
`Write` no R

Reply:
I would rather rename the property to defaultWriteMode. Not Table, no R.

Too much of `TableWriter`, that's true but this property is more in command of not using it so it would be weird to name it after the thing we want to reduce the dependency on.